client/src/components/Review/Review.jsx

```diff
@@ -78,7 +74,7 @@ class Review extends React.Component {
 
   render() {
     return (
-      <ReviewSection >
+      <div>
```

Comment:
@cchen146, to avoid polluting the DOM, we can use <React.Fragment> instead of <div> for component wrapping